templates/driver/README.md.tmpl

```diff
@@ -0,0 +1,25 @@
+# { name | upper } Driver [![crates.io](https://img.shields.io/crates/v/{ name }.svg)](https://crates.io/crates/{ name }) [![Documentation](https://docs.rs/{ name }/badge.svg)](https://docs.rs/{ name }) ![CI Build](https://github.com/braun-embedded/step-dir/workflows/CI%20Build/badge.svg)
+
+## About
+
+Rust driver crate for the [{ name | upper }] stepper motor driver. Carrier boards for this chip are [available from Pololu].
+
+This crate is a specialized facade of the [Step/Dir] library. Please consider using Step/Dir directly, as it provides drivers for more stepper motor drivers, as well as an interface to abstract over them.
+
+See [Step/Dir] for more documentation and usage examples.
+
+## Status
+
+This driver is currently very basic in its capabilities. Its design is experimental, and more revisions to the API are expected.
```

Comment:
Again, not completely on topic, but might be best to remove this. We already link to Step/Dir, so it's probably best to keep the information we need to maintain here to a reasonable minimum.